libs/transfer-journey/src/lib/transfer-journey.ts

```diff
@@ -0,0 +1,123 @@
+import { InjectionToken, Type } from '@angular/core';
+import { journeyFactory, withDefaults } from '@backbase/foundation-ang/core';
+import { Routes } from '@angular/router';
+import {
+  MakeTransferRouteTitleResolverService,
+  MakeTransferCommunicationService,
+  MakeTransferPermissionsService,
+  MakeTransferAccountHttpService,
+} from '@backbase/transfer-journey/internal/data-access';
+import { TRANSLATIONS } from '@backbase/transfer-journey/internal/shared-data';
+import { provideJourneyTracker } from '@backbase/foundation-ang/observability';
+
+// Components are directly imported since the entire journey is lazy-loaded via loadChildren
+import { TransferJourneyComponent } from './transfer-journey.component';
+import {
+  MakeTransferViewComponent,
+  MakeTransferSummaryViewComponent,
+  MakeTransferSuccessViewComponent,
+} from '@backbase/transfer-journey/internal/feature';
+import { MakeTransferJourneyStoreGuard } from './make-transfer-journey-store-guard';
+
+// Configuration Interface
+export interface MakeTransferJourneyConfig {
+  maskIndicator: boolean;
+  maxTransactionAmount: number;
+  slimMode: boolean;
+}
+
+// Default Configuration
+const defaultConfig: MakeTransferJourneyConfig = {
+  maskIndicator: true,
+  maxTransactionAmount: 100,
+  slimMode: true,
+};
+
+// Configuration Token
+export const MAKE_TRANSFER_JOURNEY_CONFIG =
+  new InjectionToken<MakeTransferJourneyConfig>('MAKE_TRANSFER_JOURNEY_CONFIG');
+
+// Communication Service Token
+export const MAKE_TRANSFER_JOURNEY_COMMUNICATION_SERVICE =
```

Comment:
I also don't see this being injected anywhere either - but I do see we're injecting the abstract `MakeTransferCommunicationService` class.

As this is a service, not a plain config value, I think it's better in this instance to delete this token and use `communicationService: MakeTransferCommunicationService` in the `journeyFactory` call below.